wiki/C++-Style-Guide.md

```diff
@@ -106,8 +106,8 @@ For example, a header file might contain the following #includes:
 class Angle;
 class Sprite;
 ```
-It is helpful but not required to alphabetize the lines within each "paragraph" of #includes.
-Forward-declaring classes instead of #including their headers reduces compilation time somewhat because it eliminates dependencies. But, compilation times are so short for this project anyways that using forward declarations is not an absolute requirement.
+It is required to alphabetize the lines within each "paragraph" of #includes and forward declarations.
```

Comment:
```suggestion
Alphabetize the lines within each "paragraph" of #includes and forward declarations.
```
Or:
```suggestion
The lines within each "paragraph" of #includes and forward declarations should be alphabetized.
```
I prefer the latter, but I think the former might be more consistent with the way the rest of the style guide is written.
It could also be "You should alphabetize [...]".
```suggestion
You should alphabetize the lines within each "paragraph" of #includes and forward declarations.
```